src/App.js

```diff
@@ -32,7 +32,7 @@ function App() {
       <Routes>
          <Route path="/" element={<Home/>} />
          <Route path="/product" element={<Product />} /> 
-         <Route path="/products" element={products.length > 0 ? <Products products={products}/> : null} />
+         {/*<Route path="/products" element={products.length > 0 ? <Products products={products}/> : null} />*/}
```

Comment:
And actually since Home has products, I just commented this out for the time being